pandas/tests/io/pytables/common.py

```diff
@@ -17,7 +17,8 @@
 def safe_remove(path):
     if path is not None:
         try:
-            ensure_clean(path)
+            with ensure_clean(path) as filename:
```

Comment:
Wasn't quite sure what to do here.

Reply:
likely have to remove 

safe_remove entirely here (in favor or ensure_clean)